.github/workflows/test_full.yml

```diff
@@ -39,16 +39,16 @@ jobs:
       - name: install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install -e .[test_full]
+          python -m pip install -e .[devel_full]
```

Comment:
In your implementation (if I understood it correctly), `test_full` includes **both** test and tutorial dependencies. Why do we switch to `devel_full` here (in `test_coverage` pipeline)?

Reply:
Reverted.